packages/ui-extensions/docs/surfaces/customer-account/reference/examples/apis/attribute-values.example.tsx

```diff
@@ -1,34 +1,16 @@
+import '@shopify/ui-extensions/preact';
 import {render} from 'preact';
-import {useEffect, useState} from 'preact/hooks';
 
 export default async () => {
-  render(<App />, document.body);
-}
-
-function App() {
-  const [
-    buyerSelectedFreeTShirt,
-    setBuyerSelectedFreeTShirt,
-  ] = useState(
-    shopify.attributes.current
-      ?.buyerSelectedFreeTShirt || false,
-  );
-  const [tshirtSize, setTshirtSize] = useState(
-    shopify.attributes.current?.tshirtSize || '',
-  );
+  render(<Extension />, document.body);
```

Comment:
Updated all examples to use `<Extension />` instead of `<App />`, since the CLI template currently uses `<Extension />`